.claude/commands/bootstrap.md

```
@@ -7,15 +7,15 @@ Set up the development environment from scratch.
 
 1. Install development tools (uv, ruff, ty, yq, etc.):
    ```bash
-   make bootstrap-tools
+   mise run setup
    ```
 
 2. Install Python dependencies (choose one):
    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)